everware/spawner.py

```diff
@@ -280,17 +278,19 @@ def start(self, image=None):
         except Exception as e:
```

Comment:
what other exceptions can occur if the building goes wrong? We should try to catch only those and let the rest pass on upwards